servicex_app/servicex_app/models.py

```diff
@@ -463,14 +463,18 @@ class DatasetFile(db.Model):
     file_events = db.Column(db.BigInteger, nullable=True)
     paths = db.Column(db.Text(), unique=False, nullable=False)
     dataset = relationship("Dataset", back_populates="files")
+    created_at = db.Column(DateTime, default=func.now())
+    updated_at = db.Column(DateTime, default=func.now(), onupdate=func.now())
```

Comment:
Not sure when we'd ever use the `updated_at` column. I think files are written once and that's the end of it